### ISSUE TITLE
fetchMock.get(string, "") fails due to "fetch-mock: Invalid parameters passed to fetch-mock"

### ISSUE THREAD

**Issue by krichter722** [wheresrhys/fetch-mock#517]
`fetchMock.get` fails due to `fetch-mock: Invalid parameters passed to fetch-mock` if the url parameter is a string and the response argument is `""`. Non-empty strings work as expected. Example:

```
import fetchMock from "fetch-mock";

afterEach(() => {
  fetchMock.reset();
});

describe('fetchMock', () => {

  it('allows returning empty response', () => {
    fetchMock.get("http://bla.blu", "");
  })
})
```

I suspect this to be a bug because the response parameter can be a string and an empty response is valid for HTTP requests. Furthermore non-empty string are returned as response as expected.

The only related issue I found is #184 which is about improving the error message. It's fine afaik, I wouldn't mind more information which types are expected, though, but that's a separate concern, and since types are provided, it's not necessary.

A complete example project is provided at https://gitlab.com/krichter-sscce/fetch-mock-empty-string-invalid-parameter. The behaviour can be triggered by running `yarn test` in the source root.

Version information from `yarn.lock`:

```
fetch-mock@^7.5.1:
  version "7.7.3"
  resolved "https://registry.yarnpkg.com/fetch-mock/-/fetch-mock-7.7.3.tgz#6a3f94cfed6e423ab7f5464912982030da605335"
  integrity sha512-I4OkK90JFQnjH8/n3HDtWxH/I6D1wrxoAM2ri+nb444jpuH3RTcgvXx2el+G20KO873W727/66T7QhOvFxNHPg==
  dependencies:
    babel-polyfill "^6.26.0"
    core-js "^2.6.9"
    glob-to-regexp "^0.4.0"
    lodash.isequal "^4.5.0"
    path-to-regexp "^2.2.1"
    whatwg-url "^6.5.0"
```

**Comment by wheresrhys** [comment]
Yes that's a bug alright.

I'll get it fixed soon.

**Comment by wheresrhys** [comment]
released as 9.2.2
